apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/CharSequenceContainsRegexAssertionsSpec.kt

```diff
@@ -32,17 +37,20 @@ class CharSequenceContainsRegexAssertionsSpec : Spek({
 
         private fun getNameContainsRegex() = "$contains with search mode $regex"
 
-        private fun getAtLeastTripleString() =
+        private fun getAtLeastTripleString(): Pair<(String, String) -> String, StringFun> =
             { what: String, times: String -> "$contains $what $atLeast $times" } to
                 ("$contains.$atLeast.$regex" to Companion::containsAtLeast)
 
-        private fun getAtLeastTripleRegex() =
+        private fun getAtLeastTripleRegex(): Pair<(Regex, String) -> String, RegexFun> =
             { what: Regex, times: String -> "$contains $what $atLeast $times" } to
                 ("$contains.$atLeast.$regex" to Companion::containsAtLeast)
 
         private fun containsAtLeast(expect: Expect<CharSequence>, atLeast: Int, a: String, aX: Array<out String>) =
             expect.contains.atLeast(atLeast).regex(a, *aX)
 
+        private fun containsAtLeast(expect: Expect<CharSequence>, atLeast: Int, a: Regex, aX: Array<out Regex>) =
```

Comment:
please do not overload it, add suffix `Regex` and `String` as you did otherwise

Reply:
The function `containsAtLeastRegex` which you could not find is the one you have to write, sorry for the confusion :grin:

Reply:
Sorry, I'm still kind of confused.
If I'm correct, I'm supposed to implement your stub like:
```kotlin
        private fun containsAtLeastString(expect: Expect<CharSequence>, atLeast: Int, a: String, aX: Array<out String>) =
            expect.contains.atLeast(atLeast).regex(a, *aX)

        private fun containsAtLeastRegex(expect: Expect<CharSequence>, atLeast: Int, a: Regex, aX: Array<out Regex>) =
            expect.contains.atLeast(atLeast).regex(a, *aX)
```
But without duplicating `CharSequenceContainsRegexAssertionsSpec` for the new Regex functions, I still get compile-time errors for
```kotlin
object RegexSpec : ch.tutteli.atrium.specs.integration.CharSequenceContainsRegexAssertionsSpec(
        getNameContainsRegex(),
        getAtLeastTripleRegex(),
        getAtLeastIgnoringCaseTripleRegex(),
        getShortcutTripleRegex(),
        getAtMostTripleRegex(),
        getAtMostIgnoringCaseTripleRegex(),
        "◆ ", "⚬ "
    )
```
because `CharSequenceContainsRegexAssertionsSpec` wants the original `String` versions.